src/lib.rs

```diff
@@ -10,6 +10,7 @@ mod header;
 pub mod models;
 #[cfg(feature = "presto")]
 mod presto_header;
+pub mod proxy;
```

Comment:
I think we can make it private so we can keep a smaller API surface.

Reply:
Definitely! I’ve made the change.